_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+---
+title: "CMORization: Using observational datasets"
```

Comment:
```suggestion
title: "CMORization: using observational datasets"
```

Reply:
I'll change the title (and take care of the capitals)

Reply:
Done